<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerTurnDialog</class>
 <widget class="QDialog" name="PlayerTurnDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Player X's turn</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>329</width>
     <height>164</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="DrunknessLabel_4">
        <property name="text">
         <string>Drunkness: 5.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="CashLabel_2">
        <property name="text">
         <string>Current cash: 100€</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="BankBalanceLabel_3">
        <property name="text">
         <string>Bank balance: 250€</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="WithdrawPushButton">
        <property name="text">
         <string>Withdraw cash</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="EnterBarPushButton_3">
        <property name="text">
         <string>Enter bar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="WalkcheckBox">
            <property name="text">
             <string>Walk</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ScootercheckBox_2">
            <property name="text">
             <string>Scooter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="NyssecheckBox_3">
            <property name="text">
             <string>Nysse</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="TaxicheckBox_4">
            <property name="text">
             <string>Taxi</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="DirectionCheckBox">
            <property name="text">
             <string>Location 1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="DirectionCheckBox_2">
            <property name="text">
             <string>Location 2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="DirectionCheckBox_3">
            <property name="text">
             <string>Location 3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="DirectionCheckBox_4">
            <property name="text">
             <string>Location 4</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="MovePushButton_2">
        <property name="text">
         <string>Move </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
